src/tests/store/activities/actions.test.js

```diff
@@ -0,0 +1,39 @@
+import { addActivity, removeActivity, editActivity } from '../../../store/activities/actions';
+import { MONDAY } from '../../../constants/dates';
+
+test('should setup add activity action object', () => {
+  const activityData = {
+    classNo: '123',
+    day: MONDAY,
+    name: 'jumping',
+    room: 'kitchen',
+    teacher: 'police man',
+  };
+  const action = addActivity(activityData);
+  expect(action).toEqual({
+    type: 'ADD_ACTIVITY',
```

Comment:
I am not sure if it is already setup, but we should keep actions names in constants.